components/brave_wallet/browser/rlp_decode.cc

```diff
@@ -85,17 +81,13 @@ bool RLPDecodeLength(const std::string& s,
     return true;
   }
 
-  if (prefix - 0xf7 < 1 ||
-      1 + prefix - 0xf7 > length || !RLPToInteger(s.substr(1, prefix - 0xf7), &i)) {
-    return false;
-  }
-  if (prefix <= 0xff && length >= prefix - 0xf7 && length >= prefix - 0xf7 + i) {
-    size_t lenOfListLen = prefix - 0xf7;
-    size_t listLen;
-    if (1 + lenOfListLen > length || !RLPToInteger(s.substr(1, lenOfListLen), &listLen)) {
-      return false;
-    }
-    *offset = 1 + lenOfListLen;
+  if (prefix - 0xf7 >= 1 &&
+      prefix <= 0xff &&
+      length >= 1 + prefix - 0xf7 &&
+      length >= prefix - 0xf7 + 1 &&
```

Comment:
These are equivalent

Reply:
Fixed by interactive rebase in the next push